### PR TITLE
Update list of downloaded songs after converted to ogg

### PR DESCRIPTION
**Pull request by taahamahdi**
ffmpeg logger improvements also

### CODE REVIEW

File: src/scripts/download-new-songs.ts [hunk 1]
```diff
@@ -151,6 +144,24 @@ const convertToOpus = async () => {
     for (const mp3File of mp3Files) {
         await ffmpegOpusJob(mp3File);
     }
+
+    let songs: Array<QueriedSong> = await db.kpopVideos("kpop_videos.app_kpop")
```
Comment (taahamahdi) [inline src/scripts/download-new-songs.ts:148]: This exists in 1 other place in this file. Should I make a getter function that takes the db and returns this?

Comment (Brainicism) [inline src/scripts/download-new-songs.ts:148]: We use this query set twice, should re-use

File: src/scripts/download-new-songs.ts [hunk 2]
```diff
@@ -151,6 +144,24 @@ const convertToOpus = async () => {
...
+        .orderBy("kpop_videos.app_kpop.views", "DESC")
+
+    //update list of non-downloaded songs
+    const songIdsNotDownloaded = songs.filter(x => !fs.existsSync(path.join(process.env.SONG_DOWNLOAD_DIR, `${x.youtubeLink}.ogg`))).map(x => ({ vlink: x.youtubeLink }));
+    await db.kmq.transaction(async (trx) => {
+        await db.kmq("not_downloaded").del().transacting(trx);
+        await db.kmq("not_downloaded").insert(songIdsNotDownloaded).transacting(trx);
+    })
+
+    await db.destroy();
```
Comment (Brainicism) [inline src/scripts/download-new-songs.ts:164]: destroy() at end of downloadAndConvertSongs

File: src/scripts/download-new-songs.ts [hunk 3]
```diff
@@ -162,6 +173,7 @@ const ffmpegOpusJob = async (mp3File: string) => {
     let oggPartWithPath = `${oggFileWithPath}.part`;
     let oggFfmpegOutputStream = fs.createWriteStream(oggPartWithPath);
 
+    logger.info(`Starting ffmpeg process for ${mp3File}`)
```
Comment (Brainicism) [inline src/scripts/download-new-songs.ts:176]: use more descriptive wording, we're encoding to ogg, mentioning ffmpeg doesn't tell us much

**Review by Brainicism — CHANGES_REQUESTED** [review]

**Review by Brainicism — APPROVED** [review]

**Pull request merged**